catroid/src/main/java/org/catrobat/catroid/ui/MainMenuActivity.java

```diff
@@ -102,6 +115,26 @@ protected void onCreate(Bundle savedInstanceState) {
 		}
 	}
 
+	private void showContentView(@Content int content) {
+		switch (content) {
+			case FRAGMENT:
+				findViewById(R.id.main_menu_buttons_container).setVisibility(View.VISIBLE);
```

Comment:
we could put all the calls to `findViewById(...)` before the switch and store the views into variables, which would make that block more readable